polar-core/src/polar.rs

```diff
@@ -5,7 +5,6 @@ use super::kb::*;
 use super::messages::*;
```

Comment:
Do you need to remove roles errors from the core also? `RolesValidationError`?

Reply:
Done in c507c66